src/main/java/com/atlantbhinternship/auctionapp/responses/LoginResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.atlantbhinternship.auctionapp.responses;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
```

Comment:
Do you need this here? Consider to remove annotations like these in other files as well?